### ISSUE TITLE
get path from node

### ISSUE THREAD

**Issue by am21** [wwkimball/yamlpath#89]
Would be really useful to get the actual path of a node. Something like this:
```
for node in processor.get_nodes(yamlpath):
    print(node.fullpath)     # returns the path that points to this node: /a/b/c[0]
```

**Comment by wwkimball** [comment]
I've opened a PR which will resolve this issue.  That PR however is part of a future release which includes other enhancements, as well, from other PRs.  As such, GitHub will automatically close this Issue once the PR is merged but you unfortunately won't be able to use the new capability until the following formal release.  I haven't yet set a schedule for the next release but I expect to roll it out sometime early next week.  I perform quite a lot of testing and write or update documentation before I release new code.

If you'd like to play with the new capability before it is released, you can certainly do so.  Just Git clone this repository, checkout the "development" branch, create a new virtual environment, activate it, and then `pip install .` to it.  As long as that virtual environment is active, you'll be using the latest development code.  Just deactivate and destroy the virtual environment when finished.

I opted to create the new attribute with the name "path" rather than "fullpath"; so the value you're after will be at `node.path`.

**Comment by am21** [comment]
Awesome, thanks! I'll give it a spin. Not in a rush for the release version, take as much time as you need 👍

**Comment by wwkimball** [comment]
I have released 3.3.0, which formally adds this capability.  Thank you for your interest in this project!
